show/portchannel.py

```diff
@@ -1,4 +1,9 @@
-#!/usr/bin/python
```

Comment:
This file is now "show/portchannel.py". If we put all the files directly under show/ we could wind up with naming collisions. I think it would be best to create a directory hierarchy for groups, such that the directory structure mirrors the group/subcommand nesting. E.g, show/interfaces/portchannel.py.